### ISSUE TITLE
Issue with template 7 of kokomi

### ISSUE THREAD

**Issue by LogCreative** [LogCreative/EnkaCard-web#1]
If the displayed character has kokomi with two or more artifacts, template 7 of [EnkaCard](https://github.com/DEViantUA/EnkaCard) will refuse to render.
```log
Traceback (most recent call last):
  File "/home/runner/work/logcreative.tech/logcreative.tech/src/assets/EnkaCard-web/main.py", line 85, in <module>
    asyncio.run(generate_cards())
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/runner/work/logcreative.tech/logcreative.tech/src/assets/EnkaCard-web/main.py", line 45, in generate_cards
    character_narrow_result = await encard.creat(ENCpy, 7)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/enkacard/encbanner.py", line 245, in creat
    result = await asyncio.gather(*task)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/enkacard/encbanner.py", line 262, in generation
    result =  await generationSeven(charter,self.assets,img, self.translateLang,self.splashArt,signatureRes,player, typelang = self.typelang) 
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/enkacard/src/utils/CreatBannerSeven.py", line 422, in generationSeven
    bg = await stats(bg, characters, assets)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/enkacard/src/utils/CreatBannerSeven.py", line 135, in stats
    AttributeBg.alpha_composite(icon,(46,posIcon[i]))
IndexError: list index out of range
Error: Process completed with exit code 1.
```

**Comment by LogCreative** [comment]
more issues detected, temporary measures are added by d050c17

**Comment by LogCreative** [comment]
It is fixed by EnkaCard 1.1.8, reverted by 4b06f34.

**Comment by LogCreative** [comment]
Solved by Enkacard 3 automatically by #3 .
